cli/flox-rust-sdk/src/models/env_registry.rs

```diff
@@ -204,14 +240,47 @@ pub struct RegisteredEnv {
     pub pointer: EnvironmentPointer,
 }
 
+/// PID of an environment's activation.
+#[derive(Debug, Clone, Serialize, Deserialize, Hash, Eq, PartialEq)]
+#[cfg_attr(test, derive(proptest_derive::Arbitrary))]
+pub struct Pid(u32);
+
+impl Pid {
+    /// Construct a Pid from the current running process.
+    pub fn from_self() -> Self {
+        Pid(nix::unistd::getpid().as_raw() as u32)
+    }
+
+    /// Check whether an activation is still running.
+    fn is_running(&self) -> bool {
+        // TODO: Compare name or check for watchdog child to see if it's a real activation?
```

Comment:
Does this todo mean to address the case where the activation is gone but another process got the pid allocated?